evm-contracts/src/v0.7/dev/Operator.sol

```diff
@@ -423,6 +403,24 @@ contract Operator is
     return _fundsAvailable();
   }
 
+  /**
+   * @notice Forward a call to another contract
+   * @dev Only callable by an authorized sender
+   * @param to address
+   * @param data to forward
+   */
+  function forward(
```

Comment:
We moved this to the `OperatorForwarder` recently. Should it also be in the `Operator`?

Reply:
We did, but this one is only callable by the owner. Maybe it would help to name it more specifically? Or maybe rename `OperatorForwarder` to `AuthorizedForwarder` since it's not really Operator specific?

Reply:
I like the idea of using a non Operator specific name. `AuthorizedForwarder` is good; we use `AccessControlled...` in some of our contracts.

Reply:
Yeah, I've been going back and forth on that. AccessControlled is slightly different, at least because the access controllers we have so far don't reset the entire access control list every time one address gets updated. It also keeps a list of all readers, in addition to a mapping. Seemed different enough that I didn't want to try and pull out an early abstraction.